prereise/gather/griddata/hifld/data_access/load.py

```diff
@@ -0,0 +1,10 @@
+import pandas as pd
+
+
+def load_csv(csv_filename, dtypes=None):
+    if dtypes:
+        data = pd.read_csv(csv_filename, dtype=dtypes)
+    else:
+        data = pd.read_csv(csv_filename)
```

Comment:
How is this behavior any different than the existing `pandas.read_csv`?

Reply:
It's not necessary, but it's a placeholder for the larger section of code as we move the data to other places. I can include the code now to read from blob storage, but I thought that was perhaps outside the scope of this PR.

Reply:
Reading from blob storage is the same, e.g,
`pd.read_csv("https://besciences.blob.core.windows.net/datasets/hifld/Electric_Substations_Jul2020.csv")`. Just the docstring would need to be modified to indicate that it can be a url instead of a file path